### PR TITLE
Create test.py to test pull request

### PR DESCRIPTION
**Pull request by Hyunmin-jasper-Cho**
## content here

### CODE REVIEW

File: test.py [hunk 1]
```diff
@@ -0,0 +1,4 @@
+import torch
+
+if __name__ == '__main__':
```
Comment (Hyunmin-jasper-Cho) [inline test.py:3]: comment to line

Comment (chromato99) [reply test.py:3]: LGTM

File: test.py [hunk 2]
```diff
@@ -2,3 +2,5 @@
 
 if __name__ == '__main__':
   print(torch.cuda.is_available())
+
+  ## test
```
Comment (chromato99) [inline test.py]: I think we can delete this annotation.

**Review by chromato99 — CHANGES_REQUESTED** [review]

File: test.py (hunk 2 above)
Comment (Hyunmin-jasper-Cho) [reply test.py]: Resolved problems

**Review by chromato99 — COMMENTED** [review]
LGTM

**Review by chromato99 — APPROVED** [review]

**Pull request merged**